### PR TITLE
Zh cn translate lite faq

### PR DESCRIPTION
**Pull request by jessie-zhao**

### CODE REVIEW

File: site/zh-cn/lite/guide/faq.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Frequently Asked Questions
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/faq.md]: 主标题不需要翻译吗

File: site/zh-cn/lite/guide/faq.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+如果这里找不到你需要的答案，请查看相关专题的详细文档或
+[GitHub issue](https://github.com/tensorflow/tensorflow/issues).
+
+## 模型转换
+
+#### 从TensorFlow到TensorFlow Lite的转换，支持哪些格式？
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/faq.md]: 中英文直接请使用空格分隔

File: site/zh-cn/lite/guide/faq.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+因为TensorFlow Lite的操作集比TensorFlow要小，一些推断模型无法被转换。对于未实现的操作，请查看
+[不支持操作](faq.md#why-are-some-operations-not-implemented-in-tensorflow-lite).
+不支持操作包括嵌入和LSTM/RNN。 对于LSTM/RNN模型，可以尝试用实验性API [OpHint](https://www.tensorflow.org/api_docs/python/tf/lite/OpHint)来转换。
+使用控制流操作（Switch、Merge等）的模型当前无法转换，我们正在TensorFlow Lite中加入控制流的支持，请看
+[GitHub issues](https://github.com/tensorflow/tensorflow/issues/28485).
+
+与不支持操作或控制流操作无关的转换相关的问题，请搜索
+[GitHub issues](https://github.com/tensorflow/tensorflow/issues?q=label%3Acomp%3Alite+)
+或开启[新的问题](https://github.com/tensorflow/tensorflow/issues).
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/faq.md]: 如果issue翻译就都翻译，要么就都不翻译

File: site/zh-cn/lite/guide/faq.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+*   *使用硬件加速器*： TensorFlow Lite通过委托（delegate）支持特定硬件下的模型加速。例如，在解释器中使用
+    [`UseNNAPI`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/interpreter.h#L343)调用Android的神经网络API。
+  请查看
+    [GPU delegate tutorial](../performance/gpu.md).
+*   *(高级) 模型调优*： The Tensorflow Lite
+    [评测工具](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/benchmark)
+    有内置的计数器，可以统计操作相关数据。如果通过它，你了解到应该优化哪个操作，你就可以用一个 [定制操作](ops_custom.md).
+
+对于更多关于性能优化的深入讨论，请查看
+[最佳实践](../performance/best_practices.md).
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/faq.md]: 中文标点

**Comment by lamberta** [comment]
Closing for inactivity. Feel free to reopen after you accept the reviewer suggestions, just make sure to rebase since things are moving around a bit now. Thank you

**Pull request closed without merging**